components/Footer.tsx

```diff
@@ -19,38 +17,6 @@ const Footer = () => {
             All rights reserved
```

Comment:
Two comments

1. IMO the "All rights reserved" should be removed. The code is available publicly and it's a project about a public book. It kinda feels wrong to leave it
2. Would appreciate if you also add my name there 🫶

Reply:
Changed